src/validations.ts

```diff
@@ -1,4 +1,5 @@
 const emailRegex = /^([a-zñ\d[\]])(\.?([\wñ&/~\-+\][]+))*@([ñ\w.\-\][]+)\.[\w\-\][.]{2,}(\?)?.*$/i;
+const nameRegex = /^[a-zA-Z][^#&<>[\](\)\"~;$^%{}?]{1,}$/g;
```

Comment:
I am not familiar with quantifiers, could you explain me how this regular expression works?

Reply:
Maybe I am wrong, but I am pretty sure that the quantifier is not necessary.

On the other hand, I am not sure that it is ok avoiding unwanted characters with a negation, because there will be always a character that you did not include, for example, an emoji.

Reply:
what recommend you??

Reply:
I am not sure, thinking...

Reply:
i updated the regex because forgot updated